spec/features/post_index_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'rails_helper'
+
+RSpec.describe 'Post', type: :feature do
+  before :each do
+    @user1 = User.create!(name: 'Bashar', photo: 'https://unsplash.com/photos/1.jpg', bio: 'Teacher from Mexico.',
+                          posts_counter: 2)
+    @post1 = Post.create!(author: @user1, title: 'First Post', text: 'First text', comments_counter: 2,
+                          likes_counter: 2)
+    @post2 = Post.create!(author: @user1, title: 'Second Post', text: 'Second text', comments_counter: 0,
+                          likes_counter: 0)
+    @comment1 = Comment.create!(post: @post1, author: @user1, text: 'Hi Bashar!')
+    @comment2 = Comment.create!(post: @post1, author: @user1, text: 'Hi Bashar!')
+    @like1 = Like.create!(post: @post1, author: @user1)
+    @like2 = Like.create!(post: @post1, author: @user1)
+  end
+  describe 'Index page' do
+    it "should display the user's profile picture" do
+      visit user_posts_path(@user1)
+      expect(page).to have_css("img[src*='https://unsplash.com/photos/1.jpg']")
+    end
+
+    it "should display the user's username" do
+      visit user_posts_path(@user1)
+      expect(page).to have_content(@user1.name)
+    end
+
+    it "should display the user's number of posts" do
+      visit user_posts_path(@user1)
+      expect(page).to have_content(@user1.posts_counter)
+    end
+
+    it "should display the post's title" do
+      visit user_posts_path(@user1)
+      expect(page).to have_content(@post1.title)
+    end
+
+    it "should display some of the post's body" do
+      visit user_posts_path(@user1)
+      expect(page).to have_content(@post1.text)
+    end
+
+    it 'should display the first comments on a post' do
+      visit user_posts_path(@user1)
+      expect(page).to have_content(@comment1.text)
+      expect(page).to have_content(@comment2.text)
+    end
+
+    it 'should display the number of comments on a post' do
+      visit user_posts_path(@user1)
+      expect(page).to have_content(@post1.comments_counter)
+    end
+
+    it 'should display the number of likes on a post' do
+      visit user_posts_path(@user1)
+      expect(page).to have_content(@post1.likes_counter)
+    end
+
+    it "should redirect me to that post's show page when clicking on the post's title" do
+      visit user_posts_path(@user1)
+      click_link @post1.title
+      expect(page).to have_current_path(user_post_path(@user1, @post1))
+    end
+
+    it 'Should display the section to create a new post' do
+      visit user_posts_path(@user1)
+      expect(page).to have_content('Create New Post')
+    end
```

Comment:
Nice job so far😉. In fulfillment of the[ requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md) for this milestone, kindly add a test that asserts that you can see a section for pagination if there are more posts than fit on the view😉.

<img width="653" alt="image" src="https://github.com/newtonah0/My_Blog1/assets/64643995/7cc1b57b-f8bf-4749-9e5d-4ab9c6a06769">